### DISCUSSION TITLE
Intermittend MQTT connection

### DISCUSSION THREAD

**Discussion by Castle1310** [rvdbreemen/OTGW-firmware#231] [Q&A]
Hi all,

I have a recent nodoshop otgw with a wifi add on. Latest firmware installed.
It connects nicely with HA thrue MQTT.

But after a while it loses its connection and HA gets the info from the OTGW intermittend.
If I look at the debug page in web ui, it styates that there is no HA connection any more. When the messages get through it states that there is a connection.

I reinstalled MQTT on HA but at this point to no avail.
Does sombody have a clue what might be the problem?

Thx,
Jelle

**Comment by rvdbreemen** [comment]
Hi @Castle1310 / Jell,

Sorry I never replied. Did you get it to work? A better place to get help is our discord community.

Hope you got it setup properly now,
Robert
